test/truefi2/TrueFiPool2.test.ts

```diff
@@ -220,13 +268,202 @@ describe('TrueFiPool2', () => {
   })
 
   describe('join-exit', () => {
-    // requires strategy
-    // requires lender
+    beforeEach(async () => {
+      await tusd.approve(pool.address, parseEth(1e7))
+      await pool.join(parseEth(1e7))
+      await tusd.mint(borrower.address, parseEth(1e6))
+      await tusd.connect(borrower).approve(pool.address, parseEth(1e6))
+    })
+
+    it('does not allow to join when joining is paused', async () => {
+      await tusd.approve(pool.address, parseEth(1e6))
+      await pool.changeJoiningPauseStatus(true)
+      await expect(pool.join(parseEth(1e6)))
+        .to.be.revertedWith('TrueFiPool: Joining the pool is paused')
+    })
+
+    it('mints liquidity tokens as 1-to-1 to TUSD for first user', async () => {
+      expect(await pool.balanceOf(owner.address)).to.equal(parseEth(1e7))
+    })
+
+    it('mints liquidity tokens proportionally to stake for next users', async () => {
+      const loan1 = await deployContract(
+        LoanToken2Factory,
+        pool.address,
+        borrower.address,
+        lender.address,
+        AddressZero,
+        parseEth(1e6),
+        DAY * 365,
+        1000,
+      )
+      await rater.mock.getResults.returns(0, 0, parseTRU(15e6))
```

Comment:
If I'm reading this right, this is 15e6 = 1e7 * 150% because we're time traveling 365/2 days (half the loan term) in this test.

Is this correct? If so, could you make the calculations (here and elsewhere) explicit? Being explicit makes the tests much more legible, if we're trying to puzzle out why it's failing months down the line!